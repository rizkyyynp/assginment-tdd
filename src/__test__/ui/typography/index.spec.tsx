import Typography from "@/components/ui/typography";
import { render } from "@testing-library/react";

describe("typography component", () => {
    it("should render typography component", () => {
        const { container } = render(<Typography type="h1">List of Users</Typography>);
        expect(container).toMatchSnapshot();
    });
});