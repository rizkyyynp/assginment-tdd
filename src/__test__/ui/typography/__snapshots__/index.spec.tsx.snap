// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`typography component should render typography component 1`] = `
<div>
  <h1>
    List of Users
  </h1>
</div>
`;
